kornia/contrib/object_detection.py

```diff
@@ -113,18 +112,18 @@ def __init__(self, size: tuple[int, int], interpolation_mode: str = "bilinear")
         self.size = size
         self.interpolation_mode = interpolation_mode
 
-    def forward(self, imgs: list[Tensor]) -> tuple[Tensor, list[ImageSize]]:
+    def forward(self, imgs: list[Tensor]) -> tuple[Tensor, Tensor]:
         # TODO: support other input formats e.g. file path, numpy
         resized_imgs, original_sizes = [], []
         for i in range(len(imgs)):
             img = imgs[i]
             # NOTE: assume that image layout is CHW
-            original_sizes.append(ImageSize(height=img.shape[1], width=img.shape[2]))
+            original_sizes.append([img.shape[1], img.shape[2]])
```

Comment:
can you specify in the docs the return order of this. This is usually the root of bugs.